lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -121,13 +123,30 @@ def #{configuration[:column]}=(position)
 
             after_commit 'remove_instance_variable(:@scope_changed) if defined?(@scope_changed)'
 
-            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{connection.quote_column_name(configuration[:column])} IS NOT NULL}) }
+            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{quoted_position_column} IS NOT NULL}) }
+
+            def self.decrement_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} - 1))
+            end
+
+            def self.increment_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} + 1))
+            end
           EOV
 
           if configuration[:add_new_at].present?
             self.send :before_create, :"add_to_list_#{configuration[:add_new_at]}"
           end
+        end
 
+        private
+
+        def update_all_with_touch(updates)
+          attrs = new.send(:timestamp_attributes_for_update_in_model)
```

Comment:
Hmm, the more I think about this, the more I actually think simply calling `.new` here is the way to go.

Since `AR::Base#timestamp_attributes_for_update_in_model` depends on `#timestamp_attributes_for_update` which is intended to be potentially overridden in the concrete class's implementation, we can't trust that the output of that method is static. It probably will be 99% of the time, but its possible that some people will experience weirdness if we cache.

Also, I'm pretty sure we can't reliably call `.new` in the `.acts_as_list` macro command, because the class definition hasn't finished executing yet, as this invocation is generally at the top.

With those drawbacks in mind, using `.new` here doesn't really seem so bad. It's definitely a bit strange, but I'm not seeing any other real downsides. I don't think it will have any performance or memory implications, because its only getting called once. Thoughts?